index.html

```diff
@@ -2,8 +2,35 @@
 <head>
   <meta name="viewport" content="width=device-width">
   <script type="module" src="./index.js"></script>
+  <style>
+    .examples {
+      margin-top: 20px;
+    }
+  </style>
 </head>
 <body>
+  <media-controller>
```

Comment:
We should kill the the media-container example below this.

Reply:
looks like I messed up when I pulled in changes from `main`, will fix that